include/rocksdb/perf_context.h

```diff
@@ -45,6 +45,11 @@ struct PerfContext {
 
   ~PerfContext();
 
+  PerfContext() {}
+
+  PerfContext(const PerfContext&);
+  PerfContext& operator=(PerfContext&);
```

Comment:
Where is the move constructor?

Reply:
Probably we should declare move constructor and move assignment as deleted? Supporting move for a variable that can potentially be thread-local seems hard to reason.

Reply:
If std::move() still works I'm OK with disabling moving destructor. If std::move() doesn't work anymore, it's better avoid the trouble of breaking users' code.

Reply:
disabling move constructor will cause std::move() to stop working, but the funny thing is with the default move constructor, the current test will pass, making me wonder if we indeed need to add a move constructor.